app/lib/max_allowed_days_loader.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+module MaxAllowedDaysLoader
+  module_function
+
+  def load_account_suspension
+    config[:account_suspension]
+  end
+
+  def load_account_inactivity
+    config[:account_inactivity]
+  end
+
+  def load_contract_unpaid_time
+    config[:contract_unpaid_time]
+  end
+
+  def config
+    @config ||= load_config
+  end
+
+  def valid_configuration?
+    config && @valid_configuration
+  end
+
+  def load_config
+    config = (ThreeScale.config.max_allowed_days || {}).each_with_object({}) do |(key, value), collection|
+      collection[key.to_sym] = value.public_send(:days) if value.to_i >= 1
+    end
+    @valid_configuration = %i[account_suspension account_inactivity contract_unpaid_time].all? { |key| config.key?(key) }
+    inform_invalid_configuration unless @valid_configuration
+    config
+  end
+
+  def inform_invalid_configuration
+    Rails.logger.warn %q(
+The feature "automatically suspend inactivy tenants and automatically schedule them for deletion" is disabled.
```

Comment:
We should make it short: `[WARNING]  Can't enable "automatic inactive tenant deletion" please revise your config"

Reply:
For this message is exactly why I pinged @thomasmaas 😄 I like your suggestion, but I will wait for him to confirm 👍

Reply:
Btw if the config lives in `settings.yml` we should probably add `enable: true|false` to it so we can disable the feature on purpose and not displaying this warning

Reply:
I'm not sure if adding an extra config just to don't show a warning is ideal... 🤔 Then maybe we can do "If `ThreeScale.config.max_allowed_days` is `nil` (then it means the config file is not there), then don't show the warning message because it means it is intended".
WDYT? @hallelujah @thomasmaas

Reply:
[WARNING] Can't enable "automatic inactive tenant account deletion" please revise your config"

Reply:
Well then if the config is not there, just do not show the warning but if it is there and incorrect show the warning